pkg/sqlqueries/SQL/mto/milestone/create_milestone_solution_link.sql

```diff
@@ -0,0 +1,25 @@
+INSERT INTO mto_milestone_solution_link (
+    id,
+    milestone_id,
+    solution_id,
+    created_by,
+    created_dts,
+    modified_by,
+    modified_dts
```

Comment:
I'd suggest only inserting created by here, and and allow the default value for created_dts, and don't set any modified by entries. This is consistent with other create statements. 

```suggestion
    created_by
```